src/dune_rules/melange/melange_rules.ml

```diff
@@ -172,6 +172,20 @@ let build_js ~loc ~dir ~pkg_name ~mode ~module_systems ~output ~obj_dir ~sctx
            ; Dep src
            ]))
 
+(* attach [deps] to the specified [alias] AND the (dune default) [all] alias.
+
+   when [alias] is not supplied, {!Melange_stanzas.Emit.implicit_alias} is
+   assumed. *)
+let add_deps_to_aliases ?(alias = Melange_stanzas.Emit.implicit_alias) ~dir deps
+    =
+  let open Memo.O in
+  let alias = Alias.make alias ~dir in
+  let dune_default_alias = Alias.all ~dir in
+  let attach alias = Rules.Produce.Alias.add_deps alias deps in
+  let+ () = attach alias
+  and+ () = attach dune_default_alias in
+  ()
```

Comment:
You could also do
```suggestion
   Memo.parallel_map ~f:attach [alias; dune_default_alias]
```

Reply:
Then you don't need to open Memo.O. Also the aliases could be inlined and the attach function written as a `~f:(fun x -> Rules.Prod ... )`. That leaves you with a one-liner.

Reply:
Good to know it! Reflected in ccff55a3a